src/build/emitter.ts

```diff
@@ -10,15 +10,11 @@ import {
   baseTypeConversionMap,
   assertUnique,
   arrayBufferViewTypes,
-} from "./helpers.js";
-import { collectLegacyNamespaceTypes } from "./legacy-namespace.js";
+} from "./helpers.ts";
+import { collectLegacyNamespaceTypes } from "./legacy-namespace.ts";
 
 /// Decide which members of a function to emit
-enum EmitScope {
-  StaticOnly,
-  InstanceOnly,
-  All,
-}
+type EmitScope = "StaticOnly" | "InstanceOnly" | "All";
```

Comment:
I'd just use an as const object and the keyof typeof trick

Reply:
Hmmmmmmmmmmmmmmmm..................... I kinda prefer this way as it's simpler and it's what we do in web APIs 🤔